frontend/src/js/ui-components/InputDate/CustomHeader.tsx

```diff
@@ -46,32 +52,37 @@ export const MonthYearLabel = styled("div")`
 
 const SelectMenu = ({
   date,
+  layout,
   options,
   onSelect,
 }: Pick<ReactDatePickerCustomHeaderProps, "date"> & {
   options: SelectOptionT[];
+  layout: "grid" | "flex";
```

Comment:
That prop confused me a little, since it's not really grid vs flex but "one column" vs "two columns", guess I'd just pass the respective OptionList component itself as a prop, but yeah, just a "nit".

Reply:
Yes you're right, I renamed the layout options